### DISCUSSION TITLE
LinkedIn is gone, So Welcome to Edgemap Discussions!

### DISCUSSION THREAD

**Discussion by resiliencetheatre** [resiliencetheatre/rpi4edgemapdisplay#4] [Announcements]
# 👋 Welcome!

I activated this discussions section just to inform everybody, that something strange happened to my LinkedIn. 

# What happened ? 

**I got this message from LinkedIn Sunday Sep 16th:**

![Screenshot from 2024-09-15 10-09-45](https://github.com/user-attachments/assets/e6b052a4-754d-476e-a68e-871972d6e6b3)

# What was that video? 

I sent [this](https://www.youtube.com/watch?v=ZWIO_Ea6z4o) Youtube video link in private chat to one contact, whom I was having discussions around helping teams in Ukraine. Trustworthy contact, referred to me by third party and whom I contacted like week ago. 

![Screenshot from 2024-09-15 10-05-56|width=200](https://github.com/user-attachments/assets/2a96b2cb-0059-40d2-9382-7e2272515a15)

# So what happens next ? 

I appealed message, contacted support and got typical '_cannot do anything, directed to another department_' answer and shortly after I lost access to whole page. I did verification with my governmental ID for LinkedIn and nothing ever since. 

# Other incidents

Just week before, I discussed in LinkedIn private chat about some capability of Edgemap and related technologies. Got asked to 'Signal group' which I genuinely believe is valid invite, but since I had no Signal on my iPhone, I did not immediately jump into that. But tried to install Signal couple of days later from App store. And you guess the rest.  iPhone gave me strange 'cannot complete' error after Signal was installed and I tried to sign in. So no Signal either for me :) 

# Conclusions 

No conclusions yet, I'll have to wait few days to see how LinkedIn sorts this out. But this is not the first time I see things going on around technologies I work with. I never thought that Edgemap and related capability would be any issue - and I still believe so. But my last summer post (in LinkedIn) about 'vetted github' is now again valid. I've noticed a lot of activity around my projects and being open source developer in these turbulent times includes clearly some challenges. 

Check back into this discussions, I'll update here, if any progress. Be also prepared to the fact that Github shuts doors as well, that's resilience!

**Comment by resiliencetheatre** [comment]
# Flu can spread..

My beautiful "Git clones" graphics is gone from Github as well. Maybe Microsoft is not that open source friendly after all.. And yes, this graph was showing some nice progress on clone volume like until this day.

![Screenshot from 2024-09-16 04-20-25](https://github.com/user-attachments/assets/00897fa1-3881-4bb2-87c7-93bb96f2309b)

**Reply by nandhinianandj** [reply]
May be it is time to move to gitlab/? I don't know, but they're a Open source first company with something almost same as github.com.>.(haven't explored deep enough to know the differences).. so may be that's better??

**Comment by nfriedly** [comment]
A while back I found that my IP had gotten flagged as a proxy in Bright cloud's database. This broke, among other things, the USPS website, and the app for my garage door opener (don't ask...)

It turns out their system [saw the word "proxy" in Nginx Proxy Manager 's default "it works!" page](https://github.com/NginxProxyManager/nginx-proxy-manager/discussions/2127#discussioncomment-6326517) (which I had set up for NextCloud, etc.), and concluded it was something nefarious.

It took about 3 months of back and forth to get to the bottom of that. The ultimate solution was to configure Nginx Proxy Manager to serve a generic 404 page for unrecognized domains, but it was a wild ride getting there.

Anyways, I hope things work out for you.

**Comment by resiliencetheatre** [comment]
You can check my LinkedIn posts also from [my fediverse](https://infosec.exchange/@retheat), so you see what this was all about.

**Comment by resiliencetheatre** [comment]
There is also [Wiki](https://resilience-theatre.com/wiki/) available at my domain. For those who left behind from LinkedIn.

**Comment by resiliencetheatre** [comment]
# Update

Account was activated again, which is great. They are not happy of my profile photo and required to change account name (from Resilience Theatre) to 'real' name. I have primary account (linked in my profile about) to my real name and thought it would be cool to have my open source project as account name on LinkedIn. And if I recall, I've seen similar approaches from others. 

I need to re-think my position on LinkedIn after this saga. I got 1k followers, gained some good discussions and was able to field my Edgemap project to very interesting domain and clearly filled void between 'dogma' and my approach. 

For now I hibernated my account, because I lost all my follows, and followers were cut by LinkedIn. I might activate it again soon, but need to form better strategy for my digital sovereignty journey. Maybe I get some advice on this. 

![Screenshot from 2024-09-16 08-44-24](https://github.com/user-attachments/assets/445321b9-670f-40c7-8614-b581ae898f06)

![Screenshot from 2024-09-16 08-45-36](https://github.com/user-attachments/assets/82b77413-73fe-4fd2-be17-716c1bdee282)

Funny enough - there is lot of going on right now and my request for 'vetted github' is actually very valid.

**Comment by leyouki** [comment]
fight, flight or freeze?
Knowing what we know about this company, I suggest its time to flight to some FOSS services.  Your work and network are too valuable to take the the risk.

Thank you for sharing.

**Comment by resiliencetheatre** [comment]
# Latest update

It surely seems that LinkedIn is screwed. I used my personal account and checked what types of accounts there are. And companies and entities have a accounts which are 'non persons' - just a company or organization name. I don't know why my Resilience Theatre was knocked down. I got email that my account is restricted again and could not log in anymore. Maybe it's time to say good bye to lot of good contacts and interactions there. 

You can use [my Github page](https://github.com/resiliencetheatre) as landing page for now, I'll update there my contact information and way to follow my progress with Edgemap related projects.

**Comment by resiliencetheatre** [comment]
# LinkedIn is Back

I got my non profit work back to LinkedIn and I have to admit, **it was all my fault**. I have understood totally wrong how I should have created that Resilience Theatre presence in LinkedIn and it caused all the trouble and pain. Funny part is that my private message containing that Meshtastic youtube link triggered this whole thing!

Now I have created [Resilience Theatre page](https://www.linkedin.com/company/resilience-theatre) and hopefully this is right approach. You are welcomed to follow me there and keeping eye on other channels which are listed on my [Github profile page](https://github.com/resiliencetheatre).
